dataflow-website/stream-developer-guides/streams/standalone-stream-sample/usage-cost-logger/pom.xml

```diff
@@ -9,9 +9,9 @@
 		<relativePath>..</relativePath>
 	</parent>
 
-	<artifactId>usage-cost-logger</artifactId>
-	<name>usage-cost-logger</name>
-	<description>Demo project for Spring Boot</description>
+	<artifactId>usage-cost-logger-${binder}</artifactId>
```

Comment:
This is what allows the binder-specific artifacts to be generated. W/o this we only have a non-binder specific named artifact that actually contains kafka libs in it. 

If this is desired then we need to also update the dev guides because they still reference the binder-specific artifacts.